feature/request-money/src/main/kotlin/org/mifospay/feature/request/money/ShowQrScreenRoute.kt

```diff
@@ -123,17 +132,7 @@ internal fun ShowQrScreen(
                 }
             }
         },
-        actions = {
-            IconButton(
-                onClick = {
-                    qrBitmap?.let {
-                        val uri = ImageUtils.saveImage(context = context, bitmap = it)
-                        shareQr(context, uri = uri)
-                    }
-                },
-            ) { Icon(MifosIcons.Share, null) }
-        },
-        modifier = modifier,
+        modifier = modifier.background(getScaffoldBackgroundColor()),
```

Comment:
lets do it here.` if (isSysteminDarktheme()) color1 else color2`
also no hardcoded colors. (line206)